concepticondata/conceptlists/Wittmann-1973-200.tsv

```diff
@@ -0,0 +1,201 @@
+Wittmann-1973-200-35	35	1247	EAR	ear	zorey	zórèy	oreille
+Wittmann-1973-200-36	36	1228	EARTH (SOIL)	earth	later	(la)tè	terre
+Wittmann-1973-200-37	37	1336	EAT	to eat	mâz-e	mâžé	manger
+Wittmann-1973-200-38	38	744	EGG	egg	dizef	zé	œuf
+Wittmann-1973-200-39	39	1248	EYE	eye	lizye	žé	œil
+Wittmann-1973-200-40	40	1280	FALL	to fall	tôb-e	tôbé	tomber
+Wittmann-1973-200-41	41	1406	FAR	far	luê	lwê	loin
+Wittmann-1973-200-42	42	323	FAT (ORGANIC SUBSTANCE)	fat-grease	lagres	grès	graisse
+Wittmann-1973-200-43	43	1217	FATHER	father	papa	papa	papa, père
+Wittmann-1973-200-44	44	1419	FEAR (BE AFRAID)	to fear	per	pè	avoir peur (de)
+Wittmann-1973-200-45	45	1201	FEATHER	feather	plim	plim	plume
+Wittmann-1973-200-46	46	1242	FEW	few	pa buku, tigin	pa âpil, kèk	peu de
+Wittmann-1973-200-47	47	1423	FIGHT	to fight	lager	gumê	lutter, se battre
+Wittmann-1973-200-48	48	221	FIRE	fire	dife	difé	feu
+Wittmann-1973-200-49	49	227	FISH	fish	posò	pwésô	poisson
+Wittmann-1973-200-50	50	493	FIVE	five	sêk	sêk	cinq
+Wittmann-1973-200-51	51	1574	FLOAT	to float	flot-e	flóté	flotter
+Wittmann-1973-200-52	52	2003	FLOW	to flow	kul-e	kulé	couler
+Wittmann-1973-200-53	53	239	FLOWER	flower	fler	flè	fleur
+Wittmann-1973-200-54	54	1441	FLY (MOVE THROUGH AIR)	to fly	âvol-e	vólé	voler
+Wittmann-1973-200-55	55	249	FOG	fog	bruyar	bruya	brouillard
+Wittmann-1973-200-56	56	1301	FOOT	foot	lipye	pié	pied
+Wittmann-1973-200-57	57	1500	FOUR	four	kat	kat(r)	quatre
+Wittmann-1973-200-58	58	1431	FREEZE	to freeze	gê fre, kôzele	žélé	geler
+Wittmann-1973-200-59	59	1507	FRUIT	fruit	fri	fwi	fruit
+Wittmann-1973-200-60	60	1447	GIVE	to give	don-e	ba(y)	donner
+Wittmann-1973-200-61	61	1035	GOOD	good	bô	bô	bon
+Wittmann-1973-200-62	62	606	GRASS	grass	lerb	zèb	herbe
+Wittmann-1973-200-63	63	1425	GREEN	green	ver	vèt	vert
+Wittmann-1973-200-64	64	1334	GUTS	guts	trip	trip	tripes, intestins
+Wittmann-1973-200-65	65	1040	HAIR	hair	seve	šivé, pwèl	cheveu(x), poil(s)
+Wittmann-1973-200-66	66	1277	HAND	hand	lamê	mê	main
+Wittmann-1973-200-67	67	1211	HE	he	li	li	lui
+Wittmann-1973-200-68	68	1256	HEAD	head	latet	tèt	tête
+Wittmann-1973-200-69	69	1408	HEAR	to hear	tâd-e	tâdé	entendre
+Wittmann-1973-200-70	70	1223	HEART	heart	leker	kè, čè	cœur
+Wittmann-1973-200-71	71	1210	HEAVY	heavy	lurd	lu	lourd
+Wittmann-1973-200-72	72	136	HERE	here	isi	isit	ici
+Wittmann-1973-200-73	73	353	STRIKE	to hit	bat-e	frapé, tapé	frapper
+Wittmann-1973-200-74	74	1448	HOLD	to hold	tini	kêbé	tenir
+Wittmann-1973-200-75	75	1239	HOW	how	kumâ, ki manyer	kômâ	comment
+Wittmann-1973-200-76	76	1435	HUNT	to hunt	lasas, pus-e	šasé	chasser
+Wittmann-1973-200-77	77	1200	HUSBAND	husband	mari, bonom	mari	mari
+Wittmann-1973-200-78	78	1209	I	I	mo-a	mwê	moi
+Wittmann-1973-200-79	79	617	ICE	ice	laglas	glas	glace
+Wittmann-1973-200-80	80	1459	IF	if	si	si	si
+Wittmann-1973-200-81	81	1460	IN	in	dâ	nâ, lâ	dans
+Wittmann-1973-200-82	82	1417	KILL	to kill	tuy-e	tuyé	tuer
+Wittmann-1973-200-83	83	1410	KNOW (SOMETHING)	to know	kon-e	kôn(ê)	connaître
```

Comment:
_connaître_ is mapped in two lists to KNOW.

Reply:
I mapped it to KNOW (SOMETHING) because that's the corresponding mapping in Swadesh-1952-200.